src/Operators/spacings_and_areas_and_volumes.jl

```diff
@@ -156,7 +158,9 @@ end
 @inline Δλᶠᵃᵃ(i, j, k, grid::LLGX) = @inbounds grid.Δλᶠᵃᵃ
 
 @inline Δφᵃᶜᵃ(i, j, k, grid::LLG)  = @inbounds grid.Δφᵃᶜᵃ[j]
+@inline Δφᵃᶜᵃ(i::AbstractArray, j::AbstractArray, k::AbstractArray, grid::LLG) = Base.stack(collect(collect(transpose(Base.stack(collect(Δφᵃᶜᵃ(1, j, 1, grid) for _ in i)))) for _ in k))
```

Comment:
It seems like there is a lot of copy and paste going on here to apply these operators to arrays. Would it be possible to maybe collect all these extra methods in one file in the `Operators` module and use metaprogramming to extend them all in case they are used with arrays. Otherwise, we need to continuously add these methods little by little.

Reply:
It's also very hard to understand the chain of 7 functions... if there is any way to clarify the code by writing it differently that would be nice.

Reply:
probably we should add a function utility 

broadcast_i_to_ijk(i, j, k, val) = Base.stack(collect(collect(transpose(Base.stack(collect(val for _ in i)))) for _ in k))

broadcast_j_to_ijk(i, j, k, val) = permutedims(Base.stack(collect(collect(transpose(Base.stack(collect(val for _ in i)))) for _ in k)), ...)

broadcast_k_to_ijk(i, j, k, val) = permutedims(Base.stack(collect(collect(transpose(Base.stack(collect(val for _ in i)))) for _ in k)), ...)


that could be called by this and all similar functions

Reply:
This is very helpful!

Are the `j` and `k` broadcast correct? They are identical.